packages/kbn-apm-synthtrace-client/src/lib/logs/index.ts

```diff
@@ -55,6 +56,12 @@ export type LogDocument = Fields &
     'error.exception.stacktrace'?: string;
     'error.log.stacktrace'?: string;
     'log.custom': Record<string, unknown>;
+    'host.geo.location': number[];
+    'host.ip': string;
+    bytes: number;
+    hour_of_day: number;
+    is_published: boolean;
+    is_failure: boolean;
```

Comment:
The PR description mentions these fields being ECS, but I couldn't find them in the spec. Is there any argument against actually using ECS fields for these?

Reply:
In case it helps, I found `boolean` fields in the [TLS field set](https://www.elastic.co/guide/en/ecs/current/ecs-tls.html).